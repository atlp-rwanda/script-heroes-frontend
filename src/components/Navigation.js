import React from 'react'
import logo from '../assets/logo.png'
import {Link} from 'react-router-dom'
import {Container, Row, Col} from 'reactstrap'

const Navigation= () =>{
    return(
        <Container fluid>
            <Row className='navigation-title'>
                <Col>
                    <h1 className='nav-title text-warning'>barefoot NOMAD</h1>
                </Col>
                <Col>
                    <ul className='center-ele'>
                    <form>
                        <select className='lang-form'>
                            <option value="english">English</option>
                            <option value="kinyarwanda">Kinyarwanda</option>
                            <option value="français">Français</option>
                        </select>
                    </form>
                    </ul>
                </Col>
            </Row>
        
            <nav className="navbar navbar-expand-lg navbar-light custom-nav">
                <Link className="navbar-brand mr-5" to='/'>
                    <img src={logo} alt='logo' className='logo'/>
                </Link>
                <button className="navbar-toggler" 
                    type="button" 
                    data-toggle="collapse" 
                    data-target="#navbarSupportedContent" 
                    aria-controls="navbarSupportedContent" 
                    aria-expanded="false" 
                    aria-label="Toggle navigation">
                    <span className="navbar-toggler-icon"></span>
                </button>
                <div className="collapse navbar-collapse" id="navbarSupportedContent">
                    <ul className="navbar-nav ml-5">
                        <li className="nav-item active">
                            <Link className="nav-link text-warning ml-4" to='/'>Home 
                                <span className="sr-only">(current)</span>
                            </Link>
                        </li>
                        <li className="nav-item">
                            <Link className="nav-link text-warning ml-4" to='/'>Travel&nbsp;Tips</Link>
                        </li>
                        <li className="nav-item">
                            <Link className="nav-link text-warning ml-4" to='/login'>About</Link>
                        </li>
                        <li className="nav-item">
                            <Link className="nav-link text-warning ml-4" to='/'>Sign&nbsp;Up</Link>
                        </li>
                        <li className="nav-item text-warning">
                            <Link className="nav-link text-warning ml-4" to='/'>Services</Link>
                        </li>
                        <li className="nav-item">
                            <Link className="nav-link text-warning ml-4" to='/'>Contact</Link>
                        </li>
                        <li className="nav-item">
                            <Link className="nav-link text-warning ml-4" to='/'>
                                <button className='book-btn'>Book&nbsp;now</button>
                            </Link>
                        </li>
                        <input
                            type='search'
                            placeholder='&#xf002;'
                            className='search-input'
                        />
                                            
                    </ul>
                </div>
            </nav>  
        </Container>
        )
}
export default Navigation
