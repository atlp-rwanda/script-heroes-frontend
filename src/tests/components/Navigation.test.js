import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'
import Navigation from '../../components/Navigation'

describe("Footer",()=>{
  it("Should match snapshot", ()=>{
    const wrapper= shallow(<Navigation/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
