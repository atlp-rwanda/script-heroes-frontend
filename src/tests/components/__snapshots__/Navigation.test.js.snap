// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should match snapshot 1`] = `
<Container
  fluid={true}
  tag="div"
>
  <Row
    className="navigation-title"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <h1
        className="nav-title text-warning"
      >
        barefoot NOMAD
      </h1>
    </Col>
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <ul
        className="center-ele"
      >
        <form>
          <select
            className="lang-form"
          >
            <option
              value="english"
            >
              English
            </option>
            <option
              value="kinyarwanda"
            >
              Kinyarwanda
            </option>
            <option
              value="français"
            >
              Français
            </option>
          </select>
        </form>
      </ul>
    </Col>
  </Row>
  <nav
    className="navbar navbar-expand-lg navbar-light custom-nav"
  >
    <Link
      className="navbar-brand mr-5"
      to="/"
    >
      <img
        alt="logo"
        className="logo"
        src="test-file-stub"
      />
    </Link>
    <button
      aria-controls="navbarSupportedContent"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarSupportedContent"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarSupportedContent"
    >
      <ul
        className="navbar-nav ml-5"
      >
        <li
          className="nav-item active"
        >
          <Link
            className="nav-link text-warning ml-4"
            to="/"
          >
            Home
            <span
              className="sr-only"
            >
              (current)
            </span>
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            className="nav-link text-warning ml-4"
            to="/"
          >
            Travel Tips
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            className="nav-link text-warning ml-4"
            to="/login"
          >
            About
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            className="nav-link text-warning ml-4"
            to="/signup"
          >
            Sign Up
          </Link>
        </li>
        <li
          className="nav-item text-warning"
        >
          <Link
            className="nav-link text-warning ml-4"
            to="/"
          >
            Services
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            className="nav-link text-warning ml-4"
            to="/"
          >
            Contact
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            className="nav-link text-warning ml-4"
            to="/profile"
          >
            Profile
          </Link>
        </li>
        <li
          className="nav-item"
        >
          <Link
            className="nav-link text-warning ml-4"
            to="/"
          >
            <button
              className="book-btn"
            >
              Book now
            </button>
          </Link>
        </li>
        <input
          className="search-input"
          placeholder=""
          type="search"
        />
      </ul>
    </div>
  </nav>
</Container>
`;
